client/lib/analytics/ad-tracking.js

```diff
@@ -6,6 +6,8 @@ import noop from 'lodash/noop';
 import some from 'lodash/some';
 import debugFactory from 'debug';
 const debug = debugFactory( 'calypso:ad-tracking' );
+import clone from 'lodash/clone';
```

Comment:
FYI. You can now import from lodash using:

```js
import { clone, cloneDeep } from 'lodash';
```

No need to change that, because it works exactly the same, but in some cases can save some typing :)